warehouse/deltalake/deltalake.go

```diff
@@ -466,6 +478,19 @@ func (dl *HandleT) getLoadFolder(tableName string, location string) (loadFolder
 	return
 }
 
+func getTableSchemaDiff(tableSchemaInUpload, tableSchemaAfterUpload warehouseutils.TableSchemaT) (diff warehouseutils.TableSchemaDiffT) {
```

Comment:
@achettyiitr Can we rename tableSchemaAfterUpload to tableSchemaInWarehouse just to be consistent ?